src/cargo/ops/tree/format/parse.rs

```diff
@@ -1,7 +1,6 @@
 //! Parser for the `--format` string for `cargo tree`.
 
-use std::iter;
-use std::str;
+use std::{iter, str};
```

Comment:
This came as a result of running `rustfmt --emit files src/cargo/ops/tree/format/mod.rs`. Technically, I hadn't meant to include any other changes than the ones in `mod.rs`, so I can update this PR to revert this file if desired.